scripts/jenkins/qat/plugin-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+#
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Deploys current jenkins build test image 'intel-qat-plugin' in the cluster.
+
+set -o pipefail
+set -o xtrace
+set -o errexit
+
+REPO_ROOT=${WORKSPACE:-$(realpath $(dirname $0)/../../..)}
+REG=${REG:-intel/}
+TAG=${TAG:-devel}
+ORG=${REG%/}
+
+if [ "${ORG}" != "intel" ] || [ "${TAG}" != "devel" ]; then
```

Comment:
this is not QAT specific. perhaps have a make target that runs this in the test makefile

Reply:
https://github.com/intel/intel-device-plugins-for-kubernetes/pull/231/files#diff-3d9fe340e7ae74dc13d12de96b2d9d72R13